src/Component/BlazorComponent/Extensions/StringExtensions.cs

```diff
@@ -15,8 +15,6 @@ public static string ReplaceFirst(this string str, string oldValue, string newVa
             return str;
         }
 
-        // todo: regex.replace is more fast?
```

Comment:
fix warnings，you can see them in review

Reply:
fix all nullable warnings? Impossible...

Reply:
The review page shows 10 only. Fix as much as you see.